docs/quick-start/set-up-byconity-dev-env-on-mac.mdx

```diff
@@ -0,0 +1,99 @@
+# ByConity MAC development environment
```

Comment:
If you use docker to set up the environment， it could work for Windows simultaneously.